### PR TITLE
Parse xTB freqs from an auxiliary file if needed

### PR DESCRIPTION
**Pull request by alongd**
Solves an issue where xTB save frequencies in a vibspectrum file instead of the primary output file

### CODE REVIEW

File: arc/parser.py [hunk 1]
```diff
@@ -127,7 +127,19 @@ def parse_frequencies(path: str,
...
-
+        if freqs.size == 0:
+            # Could not read freqs from output.out, try reading from vibspectrum
+            vibspectrum_path = os.path.join(os.path.dirname(path), 'vibspectrum')
+            if os.path.isfile(vibspectrum_path):
+                lines = _get_lines_from_file(path=vibspectrum_path)
+                for line in lines:
+                    if '$' not in line and '#' not in line:
+                        splits = line.split()
+                        if len(splits) < 5:
```
Comment (Laxzal) [inline arc/parser.py:138]: Hey Alon, what if the splits is higher than 5? Or are we expecting the splits to always be lower than 5?

Comment (alongd) [reply arc/parser.py:138]: We expect to have 5 or 6 splits, e.g.:
```
     5                       0.00         0.00000        -       - 
     6        a           4225.72         0.00000       YES     YES
```

This checks whether the num of splits is less than 5 (4 or lower), if it is then we skip the line (`continue`)

Comment (Laxzal) [reply arc/parser.py:138]: Okay thanks, just wanted to double check that we wouldn't lose any information in the if len(splits) check!

**Review by Laxzal — APPROVED** [review]
Code looks good. Also tested it from my side and the run was successful and worked as intended.

**Pull request merged**